website/docs/components/form/checkbox/partials/guidelines/guidelines.md

```diff
@@ -10,6 +10,21 @@
 - When only one choice can be selected, consider [Radio](/components/form/radio/) buttons.
 - When checking or unchecking results in an immediate change, consider [Toggle](/components/form/toggle).
 
+## Indeterminate state
+
+Indeterminate states indicate a "partially checked" condition in nested structures or tables. 
+
+
+!!! Do
```

Comment:
Love the Do/Don't illustrations, they give clear guidance. 👌

Reply:
[Suggestion] One other example that it might be useful to include (which is a common one) is in the context of a form. [Could be something like this](https://www.figma.com/file/42LK10XbP5IERhzzgMOiI2/Website-assets?type=design&node-id=2545%3A2339&mode=design&t=yLvHHraGqVWkZRDl-1)